src/routes/userRoutes.js

```diff
@@ -24,4 +24,4 @@ router.put('/user/id', User.update);
 // DELETE
 router.delete('/user/:id', User.destroy);
```

Comment:
this route is not needed

Reply:
OK.  I will also delete the destroy controller afterwards.